plaidml2/bridge/keras/__init__.py

```diff
@@ -1062,14 +1062,23 @@ def not_equal(lhs, rhs):
 def normalize_batch_in_training(x, gamma, beta, reduction_axes, epsilon=1e-3):
     I = x.tensor
     ndims = I.shape.ndims
-    if reduction_axes == None:
-        raw_axes = [ndims - 1]
+    if ndims == 4 and reduction_axes in [[0, 1, 2], [0, 2, 3]]:
+        target_shape = [1 if i in reduction_axes else x._keras_shape[i] for i in range(4)]
```

Comment:
I _think_ the `x._keras_shape` here should be replaced by `shape(x)`, but perhaps it should be replaced by `int_shape(x)` (and it's possible I'm overlooking something and we need a different approach from either of those suggestions). Do we test this code on tensors with symbolic dimensions? That would help clarify what we should do here.